### DISCUSSION TITLE
disable meow motion selection save

### DISCUSSION THREAD

**Discussion by Inc0n** [meow-edit/meow#137] [Q&A] [answered]
The motion commands that automatically select some region according to defined by each commands, for example:
- `meow-next-word`
- `meow-back-word`
- `meow-mark-word`

As the selection are being made, they also save the selection to the kill ring. 
I would want to disable this. I find it more convenient to manually save with `y`.
Is this behaviour configurable?

**Comment by DogLooksGood** [comment]
No, they will save the region to kill ring. 
Do you mean regexp-search-ring?

**Reply by DogLooksGood** [reply]
Or pasting with the middle mouse button on Linux?

**Reply by Inc0n** [reply]
Yeah, if for instance I save a region with `y` (`meow-save`), the using the motion command to navigate to the region that I want to replace with the first region, now when I do `p` (`meow-yank`). The pasted text are from the latest selection, which is not I wanted.

**Reply by DogLooksGood** [reply]
That's weird, it shouldn't behave like what you describe.

1. what's is the command bound to `C-y` and `M-w` in your case? it should be `yank` and `kill-ring-save`.
2. To replace the region, you use `r (meow-replace)`

The yank command will not paste the selection. And the command on `C-y` is used as underlying command by `meow-yank`.  If that command will paste the selection, that could be problem.

**Reply by Inc0n** [reply]
I see I have left this discussion untended, my apologies. Now I am back to `meow`, this issue is once again prevalent.

I did some more digging and as it turns out it is an issue (mis-configuration) with emacs itself, i.e. not caused by `meow`.

**Comment by Inc0n** [comment, marked as answer]
As it turns out it is an issue with emacs itself, i.e. not caused by `meow`.
The reason is because I had `select-enable-primary` set to `t`.
See this reddit comment for more information: https://www.reddit.com/r/emacs/comments/5lybts/comment/dc0ut9e/
